### PR TITLE
Remove Forms from handler samples

### PR DESCRIPTION
**Pull request by PureWeen**
### Description of Change ###

Remove Xamarin.Forms from the Handler Samples. The Handler Samples should use concrete implementations that are as simple as possible

### Testing Procedure ###
- Sample App runs

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: src/Platform.Handlers/samples/Sample/Controls/Slider.cs [hunk 1]
```diff
@@ -81,5 +80,41 @@ void ISlider.DragCompleted()
 				DragCompleted?.Invoke();
 			}
 		}
+
+		public static double Clamp(double self, double min, double max)
```
Comment (PureWeen) [inline src/Platform.Handlers/samples/Sample/Controls/Slider.cs:84]: These are internal inside our handler code. I didn't want to just make those public willy nilly

File: src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.Android.cs [hunk 2]
```diff
@@ -26,13 +26,22 @@ public void SetFrame(Rectangle frame)
...
 
+
 			var left = Context.ToPixels(frame.Left);
 			var top = Context.ToPixels(frame.Top);
 			var bottom = Context.ToPixels(frame.Bottom);
 			var right = Context.ToPixels(frame.Right);
 			var width = Context.ToPixels(frame.Width);
 			var height = Context.ToPixels(frame.Height);
 
+			if (nativeView.Bottom == bottom &&
```
Comment (PureWeen) [inline src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.Android.cs:37]: Without this check the call to layout that used the same parameters caused a StackOverflowException

Comment (PureWeen) [reply src/Platform.Handlers/src/Xamarin.Platform.Handlers/Handlers/View/AbstractViewHandler.Android.cs:37]: @hartez  FYI To this code I had to do to make this PR work. Even we don't merge this PR this seems relevant

**Review by rmarinho — APPROVED** [review]

**Comment by rmarinho** [comment]
We should create a sample under Maui.Controls to test the shims / and how forms behaves

**Pull request closed without merging**